### PR TITLE
zh-trans:/docs/concepts/overview/working-with-objects/names.md

### PR DESCRIPTION
**Pull request by ping035627**
zh-trans:/docs/concepts/overview/working-with-objects/names.md.

### CODE REVIEW

File: content/zh/docs/concepts/overview/working-with-objects/names.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+<!--
+All objects in the Kubernetes REST API are unambiguously identified by a Name and a UID.
+-->
+Kubernetes REST API中的所有对象都由名称和UID明确标识。
```
Comment (xiangpengzhao) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]: 中英文之间增加一下空格，下同。

File: content/zh/docs/concepts/overview/working-with-objects/names.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+<!--
+For non-unique user-provided attributes, Kubernetes provides [labels](/docs/user-guide/labels) and [annotations](/docs/concepts/overview/working-with-objects/annotations/).
+
+See the [identifiers design doc](https://git.k8s.io/community/contributors/design-proposals/architecture/identifiers.md) for the precise syntax rules for Names and UIDs.
+-->
+对于非唯一的用户提供的属性，Kubernetes提供了[标签](/docs/user-guide/labels)和[注释](/docs/concepts/overview/working-with-objects/annotations/)。
```
Comment (xiangpengzhao) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]: 引用的链接使用中文文档中对应的链接。

File: content/zh/docs/concepts/overview/working-with-objects/names.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+<!--
+By convention, the names of Kubernetes resources should be up to maximum length of 253 characters and consist of lower case alphanumeric characters, `-`, and `.`, but certain resources have more specific restrictions.
+-->
+按照惯例，Kubernetes资源的名称最大长度应为253个字符，由小写字母、数字、`-`和`.`组成，但某些资源有更具体的限制。
+
+<!--
+For example, here’s the configuration file with a Pod name as `nginx-demo` and a Container name as `nginx`:
+-->
+例如，下面是一个配置文件，Pod名为“nginx demo”，容器名为“nginx”：
```
Comment (xiangpengzhao) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]: 原文中 `nginx demo` 和 `nginx` 引用使用的是反撇符号

File: content/zh/docs/concepts/overview/working-with-objects/names.md (hunk 1 above)
Comment (xichengliudui) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]:
```suggestion
Kubernetes REST API 中的所有对象都由名称和 UID 明确标识。
```

File: content/zh/docs/concepts/overview/working-with-objects/names.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+Kubernetes REST API中的所有对象都由名称和UID明确标识。
+
+<!--
+For non-unique user-provided attributes, Kubernetes provides [labels](/docs/user-guide/labels) and [annotations](/docs/concepts/overview/working-with-objects/annotations/).
+
+See the [identifiers design doc](https://git.k8s.io/community/contributors/design-proposals/architecture/identifiers.md) for the precise syntax rules for Names and UIDs.
+-->
+对于非唯一的用户提供的属性，Kubernetes提供了[标签](/docs/user-guide/labels)和[注释](/docs/concepts/overview/working-with-objects/annotations/)。
+
+有关名称和UIDs的精确语法规则，请参见[标识符设计文档](https://git.k8s.io/community/contributors/design-proposals/architecture/identifiers.md)。
```
Comment (xichengliudui) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]:
```suggestion
有关名称和 UID 的精确语法规则，请参见[标识符设计文档](https://git.k8s.io/community/contributors/design-proposals/architecture/identifiers.md)。
```

File: content/zh/docs/concepts/overview/working-with-objects/names.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+## Names1
+-->
+## 名称
+
+{{< glossary_definition term_id="name" length="all" >}}
+
+<!--
+By convention, the names of Kubernetes resources should be up to maximum length of 253 characters and consist of lower case alphanumeric characters, `-`, and `.`, but certain resources have more specific restrictions.
+-->
+按照惯例，Kubernetes资源的名称最大长度应为253个字符，由小写字母、数字、`-`和`.`组成，但某些资源有更具体的限制。
```
Comment (xichengliudui) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]:
```suggestion
按照惯例，Kubernetes 资源的名称最大长度应为 253 个字符，由小写字母、数字、`-` 和 `.` 组成，但某些资源有更具体的限制。
```

File: content/zh/docs/concepts/overview/working-with-objects/names.md (hunk 3 above)
Comment (xichengliudui) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]:
```suggestion
例如，下面是一个配置文件，Pod 名为 `nginx demo`，容器名为 `nginx`：
```

**Comment by xichengliudui** [comment]
@ping035627 are you still working for this pr?

**Comment by ping035627** [comment]
@xichengliudui yes, I have changed in your opinion.
@xiangpengzhao

**Comment by xichengliudui** [comment]
Yeah, great! Can you build locally first to verify why the build is currently failing and fix it
build ref: https://github.com/k8smeetup/k8s-official-translation/blob/master/local-build.md
/approve

**Comment by ping035627** [comment]
@xichengliudui fixed over, need lgtm, thanks.

**Comment by xichengliudui** [comment]
nice!
/lgtm
/approve

File: content/zh/docs/concepts/overview/working-with-objects/names.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+Kubernetes REST API 中的所有对象都由名称和 UID 明确标识。
+
+<!--
+For non-unique user-provided attributes, Kubernetes provides [labels](/docs/user-guide/labels) and [annotations](/docs/concepts/overview/working-with-objects/annotations/).
+
+See the [identifiers design doc](https://git.k8s.io/community/contributors/design-proposals/architecture/identifiers.md) for the precise syntax rules for Names and UIDs.
+-->
+
+对于非唯一的用户提供的属性，Kubernetes 提供了[标签](/docs/user-guide/labels)和[注释](/docs/concepts/overview/working-with-objects/annotations/)。
```
Comment (xichengliudui) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]: @ping035627 have you tried these two links? Seems to be 404？

Comment (ping035627) [reply content/zh/docs/concepts/overview/working-with-objects/names.md]: 英文文档中 /docs/user-guide 目录已经删掉了，该目录下的文件用脚本做了统一转换，/docs/user-guide/labels实际应该是/docs/concepts/overview/working-with-objects/labels，但无论哪个，中文文档中都还没有被翻译，所以肯定访问不了。

**Pull request merged**